packages/host/app/resources/room.ts

```diff
@@ -167,8 +170,9 @@ export class RoomResource extends Resource<Args> {
         continue;
       }
       let cardId = cardDoc.data.id;
+      let loadingPromise;
       if (!this._skillCardsCache.has(cardId)) {
-        this.cardService
+        loadingPromise = this.cardService
```

Comment:
Is there a better way of waiting for this all to be "settled"? Either existing or a better common setup?